docs/quick-start.md

```diff
@@ -1,28 +1,176 @@
+   ```
+   
+2. Send the data to the pod via curl
+   ```
+   curl -kq -X POST https://localhost:8443/vertices/in -d '{"data":[0.9,0.1,0.2,0.9,0.9,0.9,0.9,0.8,1,0.9,0.9,0.7]}'
+   ```
+   Note: only send an array of length 12 in data, as the sequence length used for training is 12.   
+
+   
+### Training
 
-## Using Jupyter Notebook:
+Initially there is no ML model present, to trigger training do a curl command and send any data to the pipeline. 
+
+The following logs will be seen in the training pod.
 
-Start Jupyter Notebook, navigate to the examples folder and open the quick start Jupyter notebook. You can play with different available options for each step.
 ```shell
-jupyter Notebook .
-```
\ No newline at end of file
+> curl -kq -X POST https://localhost:8443/vertices/in -d '{"data":[0.9,0.1,0.2,0.9,0.9,0.9,0.9,0.8,1,0.9,0.9,0.7]}'
+
+> kubectl logs numalogic-simple-pipeline-train-0-xxxxx -c udf
+2022-10-19 22:38:45,431 - INFO - Training autoencoder model..
+2022-10-19 22:38:45,783 - INFO - epoch : 5, loss_mean : 0.0744678
+2022-10-19 22:38:46,431 - INFO - epoch : 10, loss_mean : 0.0491540
+...
+2022-10-19 22:38:49,645 - INFO - epoch : 95, loss_mean : 0.0047888
+2022-10-19 22:38:49,878 - INFO - epoch : 100, loss_mean : 0.0043651
+2022-10-19 22:38:49,880 - INFO - 8b597791-b8a3-41b0-8375-47e168887c54 - Training complete
+Successfully registered model 'ae::model'.
+2022/10/19 22:38:52 INFO mlflow.tracking._model_registry.client: Waiting up to 300 seconds for model version to finish creation.                     Model name: ae::model, version 1
+Created version '1' of model 'ae::model'.
+2022-10-19 22:38:52,920 - INFO - 8b597791-b8a3-41b0-8375-47e168887c54 - Model Saving complete
+```
+
+### Inference
+
+Now, the pipeline is ready for inference with the model trained above, data can be sent to the pipeline for ML inference. 
+
+In the output, `ts_data` is the final array that the input array has been transformed to, after all the steps in the pipeline. `anomaly_score` is the final anomaly score generated for the input data.
+
+Sending non-anomalous data: 
+```
+curl -kq -X POST https://localhost:8443/vertices/in -d '{"data":[358.060687,326.253469,329.023996,346.168602,339.511273,359.080987,341.036110,333.584121,376.034150,351.065394,355.379422,333.347769]}'
+
+2022/10/20 04:54:44 (out) {"ts_data": [[0.14472376660734326], [0.638373062689151], [0.8480656378656608], [0.4205087588581154], [1.285475729481929], [0.8136729095134241], [0.09972157219780131], [0.2856860200353754], [0.6005371351085002], [0.021966491476278518], [0.10405302543443251], [0.6428168263777302]], "anomaly_score": 0.49173648784304, "uuid": "0506b380-4565-405c-a3a3-ddc3a19e0bb4"}
+```
+
+Sending anomalous data:
+```
+curl -kq -X POST https://localhost:8443/vertices/in -d '{"data":[358.060687,326.253469,329.023996,346.168602,339.511273,800.162220,614.091646,537.250124,776.034150,751.065394,700.379422,733.347769]}'
+
```

Comment:
call out that the logs are from the output vertex. this looks like `curl`'s response.

put the `get logs` command

Reply:
Got it